src/bun.js/bindings/NodeVM.cpp

```diff
@@ -1608,4 +1768,40 @@ static String stringifyAnonymousFunction(JSGlobalObject* globalObject, const Arg
     return program;
 }
 
+static RefPtr<JSC::CachedBytecode> getBytecode(JSGlobalObject* globalObject, JSC::ProgramExecutable* executable, JSC::SourceCode source)
+{
+    auto& vm = JSC::getVM(globalObject);
+    JSC::CodeCache* cache = vm.codeCache();
+    JSC::ParserError parserError;
+    JSC::UnlinkedProgramCodeBlock* unlinked = cache->getUnlinkedProgramCodeBlock(vm, executable, source, {}, parserError);
+    if (!unlinked) {
+        return nullptr;
+    }
```

Comment:
Should we check `if (parserError.isValid()`? I know that bytecode generation can error in `getUnlinkedModuleProgramCodeBlock`, likely we need to check and throw here too